src/Compatibility/Maps/src/iOS/MapRenderer.cs

```diff
@@ -342,15 +343,15 @@ void OnMapClicked(UITapGestureRecognizer recognizer)
 
 			var tapPoint = recognizer.LocationInView(Control);
 			var tapGPS = ((MKMapView)Control).ConvertPoint(tapPoint, Control);
-			((Map)Element).SendMapClicked(new Position(tapGPS.Latitude, tapGPS.Longitude));
+			((IMap)Element).Clicked(new Devices.Sensors.Location(tapGPS.Latitude, tapGPS.Longitude));
 		}
 #endif
 
 		void UpdateRegion()
 		{
 			if (_shouldUpdateRegion)
 			{
-				MoveToRegion(((Map)Element).LastMoveToRegion, false);
+			//	MoveToRegion(((Map)Element).LastMoveToRegion, false);
```

Comment:
Intentionally left commented?

Reply:
Yes we removed this api, but i want to leave the code there if we need to check something on the "old renderer"